res/values/setup.xml

```diff
@@ -30,7 +30,7 @@
     <!-- Colors -->
     <color name="login_background_color">#FFFFFF</color>
     <color name="login_logo_background_color">#FFFFFF</color>
-    <color name="background_color">#F7F7F7</color>
+    <color name="background_color">#FFFFFF</color>
```

Comment:
Is there any special reason for this change? I cannot see any special change in the appereance.